extensions/azurecore/package.json

```diff
@@ -128,6 +128,133 @@
               "Microsoft Authentication Library"
             ],
             "deprecationMessage": "Warning: ADAL has been deprecated, and is scheduled to be removed in a future release. Please use MSAL (default option) instead."
+          },
+          "azure.providerSettingsJson": {
```

Comment:
Maybe just `customProviderSettings`? The JSON part is kind of redundant, and specifying that it's custom settings would probably make this more clear